examples/language/llama/benchmark.py

```diff
@@ -227,6 +255,8 @@ def empty_init():
             overlap_allgather=args.overlap_allgather,
             use_fp8=args.use_fp8,
             fp8_communication=args.use_fp8_comm,
+            scheduler_nodes=scheduler_nodes,
+            make_vocab_size_divisible_by=1,
```

Comment:
what's this problem?

Reply:
> what's this problem?

Fixed in https://github.com/hpcaitech/ColossalAI/pull/6083/commits/982e4ee1f85a3aec285197d8bbdc3ca292bd15ff;
Use a default make_vocab_size_divisible_by=64 now;